GitDepend/Busi/Nuget.cs

```diff
@@ -62,10 +62,17 @@ private ReturnCode ExecuteNuGetCommand(string arguments)
                 .ToArray();
 
             var code = NuGet.CommandLine.Program.Main(args);
-
+            var output = sb.ToString();
+            var hasWarnings = output.ToLower().Contains("warning");
+            
             Console.SetOut(oldOut);
             
             _console.WriteLine($"nuget {arguments}");
+            if (hasWarnings)
+            {
+                _console.WriteLine(output);
+                code = 3;
```

Comment:
don't use a magic number here. Use an appropriate `ReturnCode`. If none exists, please add one, and create the appropriate string in the resource file.